### PR TITLE
Add three dot image for cache popup (Fix #7251)

### LINKED ISSUE #7251: Three dot icon replaced by new icon on cache popup
##### Detailed steps causing the problem:
- Open map
- Click on a cache on the map to open the popup

##### Actual behavior after performing these steps:
The popup "action bar" now contains a new icon instead of the three-dot icon

##### Expected behavior after performing these steps:
Actually the new icon is pretty same style as the other icons.
So instead of havin the three-dot back here, I could also imagine to keep it but then maybe make it consistent in other activities.

##### Version of c:geo used:
2019.01.23-NB

##### Is the problem reproducible for you?
Yes

##### System information:
Android 8.0

##### Other comments and remarks:
Picture from @okainov

### PR DESCRIPTION
**Pull request by bekuno**
Add three dot image as vector image
Fix #7251 

Details:
- three_dot_vertical_vec.xml
	- the icon as xml
- actionbar_popup
	- change the image to the vector image three_dot_vertical_vec (app:srcCompat)
	- add header line for xmlns:app
	- set longClickable="false" to false for 2 ImageViews without long click action
	- minor formatting
- strings_not_traslateable
	- row for the image source: material.io, Apache license
- build gradle
	- use support library for vector images in android API below 21
	- minor formatting

### CODE REVIEW

**Comment by Lineflyer** [comment]
retest this please

**Comment by Lineflyer** [comment]
retest this please

**Comment by Lineflyer** [comment]
retest this please

**Comment by Lineflyer** [comment]
retest this again please

**Comment by Lineflyer** [comment]
retest this please

**Comment by Lineflyer** [comment]
Setting "Do not merge" for now to keep better overview of open PR as the CI has problems with this PR and it is also not building for @rsudev

**Comment by bekuno** [comment]
If the compile problems not resolveable the I would change to integrate static images.

**Comment by bekuno** [comment]
now ready to merge

File: main/res/layout/actionbar_popup.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
     <ImageView
         android:id="@+id/setAsTarget"
         style="@style/action_bar_action"
-        android:longClickable="true"
+        android:longClickable="false"
```
Comment (UniQP) [inline main/res/layout/actionbar_popup.xml]: This looks unrelated to #7251.

File: main/res/layout/actionbar_popup.xml [hunk 2]
```diff
@@ -35,8 +35,7 @@
     <ImageView
         android:id="@+id/overflowActionBar"
         style="@style/action_bar_action"
-        android:longClickable="true"
-        android:src="@android:drawable/ic_menu_more" />
-
+        android:longClickable="false"
```
Comment (UniQP) [inline main/res/layout/actionbar_popup.xml]: This looks unrelated to #7251.

Comment (bekuno) [reply main/res/layout/actionbar_popup.xml]: You are right.
I will create a separate PR.

Comment (Lineflyer) [reply main/res/layout/actionbar_popup.xml]: Ups...to late I guess. Anything regarding the longClick we should now, why it was changed.

**Comment by Lineflyer** [comment]
I reverted the merging as I have seen that the coloring of the button is not ok for me.
(I hope, that Github reverts correctly...never used this feature)

grafik

**Comment by bekuno** [comment]
Very strange.
I checked the images in this PR - they are grey as expected.
Which resolution has your device, and which android version?
